src/main/java/edu/kis/powp/jobs2d/drivers/macro/MacroDriver.java

```diff
@@ -0,0 +1,55 @@
+package edu.kis.powp.jobs2d.drivers.macro;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.OperateToCommand;
+import edu.kis.powp.jobs2d.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.features.DriverFeature;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class MacroDriver implements Job2dDriver {
```

Comment:
You should mention the name of the design pattern used here (tip: a driver field inside a driver).